types/akamai-edgeworkers/test/url-search-params-tests.ts

```diff
@@ -0,0 +1,63 @@
+import URLSearchParams from 'akamai-edgeworkers/url-search-params';
```

Comment:
This doesn't match how this module is referenced from the akamai edgeworkers samples: https://github.com/akamai/edgeworkers-examples/blob/f273d82ca7186e08345ab65d383db4e723a03282/conference-details/main.js#L8